plugins/modules/docker_compose_v2.py

```diff
@@ -0,0 +1,923 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright Léo El Amri
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = '''
+---
+module: docker_compose_v2
+
+short_description: Manage multi-container Docker applications with Docker Compose.
+
+description:
+  - Uses Docker Compose to start and shutdown services.
+  - Swarm mode is not supported (thus secrets and configs are not supported).
+  - Configuration can be read from a Compose file or inline using the I(definition) option.
+  - See the examples for more details.
+
+attributes:
+  check_mode:
+    support: none
+    description: Check mode is not supported because the `docker-compose` CLI doesn't allow it.
+  diff_mode:
+    support: none
+    description: Diff mode is not supported.
+
+options:
+  docker_host:
+    description:
+      - The URL or Unix socket path used to connect to the Docker API. To connect to a remote host, provide the
+        TCP connection string. For example, C(tcp://192.0.2.23:2376). If TLS is used to encrypt the connection,
+        the module will automatically replace C(tcp) in the connection URL with C(https).
+      - If the value is not specified in the task, the value of environment variable C(DOCKER_HOST) will be used
+        instead. If the environment variable is not set, the default value will be used.
+    type: str
+    default: unix://var/run/docker.sock
+    aliases: [ docker_url ]
+  project_src:
+    description:
+      - Path to the root directory of the Compose project.
+      - Required when I(definition) is provided.
```

Comment:
I don't see an option called `definition`.

Reply:
There is one :
* https://github.com/ansible-collections/community.docker/blob/7c109a366a6a507f9d9aa82a95f24ca19a40c21b/plugins/modules/docker_compose_v2.py#L94
* https://github.com/ansible-collections/community.docker/blob/7c109a366a6a507f9d9aa82a95f24ca19a40c21b/plugins/modules/docker_compose_v2.py#L737